### ISSUE TITLE
Detailing installation instructions

### ISSUE THREAD

**Issue by ThomasTJdev** [KeepCoolWithCoolidge/nimlibxlsxwriter#1]
When using the wrapper after `nimble install nimlibxlsxwriter`, I got the error: 
**`Error: cannot open 'xlsxwriter'`**

I had to look into the nimble packages to figure out, I needed to run:
`nimble install nimlibxlsxwriter`
`nimble setup nimlibxlsxwriter`

To ease the installation for other users, you could:
1. Add this information to the `README.md`
2. Autorun the setup

What does `setupTask()` in the `.nimble`? I did not execute.
```nim
before install:
    setupTask()
```

**Comment by genotrance** [comment]
There's an issue with nimble where the `before install` task does not run. I have a [PR](https://github.com/nim-lang/nimble/pull/464) in place but it's awaiting a review.

In the meantime, you can:

```
git clone https://github.com/KeepCoolWithCoolidge/nimlibxlsxwriter
cd nimlibxlsxwriter
nimble install -y
```

**Comment by genotrance** [comment]
Please note the thread to fix this in 0.19.0.

https://github.com/nim-lang/nimble/issues/549

You need to use `after install` and `before uninstall` going forward due to changes in how nimble works.
